examples/cs/core/inheritance/program.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Examples
+{
+    using System;
+    using System.Diagnostics;
+    using Bond;
+    using Bond.Protocols;
+    using Bond.IO.Unsafe;
+
+    using examples.inheritance;
+
+    static class Program
+    {
+        static void Main()
+        {
+            var derivedObj = new Derived {str = "derived"};
+            ((Base) derivedObj).str = "derived base";
+            var derivedPayload = new CompactBinaryReader<InputBuffer>(
+                new InputBuffer(Serialize(derivedObj)));
+
+            {
+                // Deserialize Base from payload containing Derived
+                Base obj = Deserialize<Base>.From(derivedPayload);
+                Debug.Assert(Comparer.Equal<Base>(obj, derivedObj));
+            }
+        }
+
+        static ArraySegment<byte> Serialize<T>(T obj)
+        {
+            var output = new OutputBuffer(128);
```

Comment:
Why not just leave it to the default? 

As this is an example code it may create the impression that 128 is the recommended initial capacity (if that is in indeed the case then it should appear in OutputBuffer code).